pennylane/ops/qubit/special_unitary.py

```diff
@@ -298,32 +434,240 @@ def compute_matrix(theta, num_wires):
             A = qml.math.tensordot(theta, pauli_basis_matrices(num_wires), axes=[[-1], [0]])
         return qml.math.expm(1j * A)
 
-    @staticmethod
-    def compute_decomposition(theta, wires, num_wires):
-        r"""Representation of the operator as a product of other operators (static method).
+    def get_one_parameter_generators(self, interface=None):
+        r"""Compute the generators of one-parameter groups that reproduce
+        the partial derivatives of a special unitary gate.
 
-        .. math:: O = O_1 O_2 \dots O_n.
+        Args:
+            interface (str): The auto-differentiation framework to be used for the
+                computation.
 
-        This ``Operation`` is decomposed into the corresponding ``QubitUnitary``.
+        Returns:
+            tensor_like: The generators for one-parameter groups that reproduce the
+            partial derivatives of the special unitary gate.
+            There are :math:`d=4^n-1` generators for :math:`n` qubits, so that the
+            output shape is ``(4**num_wires-1, 2**num_wires, 2**num_wires)``.
+
+        Consider a special unitary gate parametrized in the following way:
+
+        .. math::
+
+            U(\theta) &= \exp\{A(\theta)\}\\
+            A(\theta) &= \sum_{m=1}^d i \theta_m P_m\\
+            P_m &\in \{I, X, Y, Z\}^{\otimes n} \setminus \{I^{\otimes n}\}
+
+        Then the partial derivatives of the gate can be shown to be given by
+
+        .. math::
+
+            \frac{\partial}{\partial \theta_\ell} U(\theta) &= U(\theta)
+            \frac{\mathrm{d}}{\mathrm{d}t}\exp\left(t\Omega_\ell(\theta)\right)\large|_{t=0}
+
```

Comment:
Not rendering correctly

Reply:
Thanks! I fixed this before but reintroduced the error :facepalm: